### ISSUE TITLE
Thanks to your code

### ISSUE THREAD

**Issue by Roger672** [mali19064/LSTM-CRF-pytorch-faster#2]
Thanks to your code.
I trained a NER model with LSTM_CRF_faster_parallel.py. During the trainning, the loss values could be negetive in some steps. So I changed the loss into 
torch.sum(torch.abs(forward_score - gold_score))/batch_size.
Then the loss values  were all positive and the model was better fitting with my data.

**Comment by mali19064** [comment]
Thanks for your feedback, the problem has been repaired now,
